### PR TITLE
add PageLink component

### LINKED ISSUE #25: build Link component
build a 'root' Link component with variants for the different Links in the app.

image

image

image

------
- [x] build root component
- [x] 'addNew' link
- [x] 'about' link
- [x] 'plus' link

### PR DESCRIPTION
**Pull request by jujuzitroni**
fixes #25 

heroku:
https://plants-need-links-egr2zl3rxrol.herokuapp.com/storybook/?path=/story/component-pagelink--plus-link

### CODE REVIEW

**Review by daebert — APPROVED** [review]
LGTM!

File: src/app/components/Icons/PlusIcon.tsx [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+function PlusLabel(): JSX.Element {
+  return (
+    <svg width="36" height="36" viewBox="0 0 36 36">
+      <g transform="translate(-55 -696.144)">
+        <line
+          y2="26.188"
+          transform="translate(73.073 700.144)"
+          fill="none"
+          stroke="#fff"
```
Comment (lajanzen) [inline src/app/components/Icons/PlusIcon.tsx:11]: Maybe you have a color variable for this?

**Review by lajanzen — APPROVED** [review]
🚀

**Review by dhnazarenko — COMMENTED** [review]
LGTM 🚀

**Pull request merged**